packages/legend-studio/src/index.ts

```diff
@@ -41,6 +41,8 @@ export {
 } from './stores/shared/DnDUtil';
 export { ExplorerTreeRootPackageLabel } from './stores/ExplorerTreeState';
 export * from './stores/DSLMapping_LegendStudioPlugin_Extension';
+export { GenerationFile } from './stores/shared/FileGenerationTreeUtil';
```

Comment:
Could you move these down to the the last block `TO BE MODULARIZED`? thanks!

Reply:
Fixed.